src/ritika/Asgmt_20_Pg_3and4.java

```diff
@@ -0,0 +1,38 @@
+package ritika;
+/* Assignment-20 :
+
+Program : 3
+swap 2 String with using 3rd variable
+
+Program : 4
+swap 2 String without using 3rd variable
+Hint : use SubString method of String class
+*/
+public class Asgmt_20_Pg_3and4 {
+	void swapStringUsing3rdVar(String s1, String s2) {
+		System.out.println("Strings at position 1 and 2 respectively are: " +s1 +" and "+s2);
+		String temp = s1;
+		s1 = s2;
+		s2=temp;
+		System.out.println("Strings after swapping at position 1 and 2 respectively are: " +s1 +" and "+s2);
+	}
+	
+	void swapStringWithout3rdVar(String s1, String s2) {
+		System.out.println("Strings at position 1 and 2 respectively are: " +s1 +" and "+s2);
+		s1 = s1+" "+s2;
```

Comment:
Very Good @RitikaGOct2020 
but think if some pass two words with space that this will not code.
correct?
Small correction require, pls refer yogita's code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/yogita/SwapString.java